Enable clipboard integration by default, can be turned of with -noclipboard.
We still accept -clipboard for backwards compatibility.  If both are passed,
the last one is accepted (just as other arguments are handled).

--- origsrc/xorg-server-1.6.1.901/hw/xwin/InitOutput.c	2009-06-24 20:36:30.751261100 -0500
+++ src/xorg-server-1.6.1.901/hw/xwin/InitOutput.c	2009-06-25 14:43:47.350612500 -0500
@@ -809,9 +809,8 @@ winUseMsg (void)
 	  "\tmonitors are present.\n");
 
 #ifdef XWIN_CLIPBOARD
-  ErrorF ("-clipboard\n"
-	  "\tRun the clipboard integration module.\n"
-	  "\tDo not use at the same time as 'xwinclip'.\n");
+  ErrorF ("-[no]clipboard\n"
+	  "\tEnable [disable] the clipboard integration. Default is enabled.\n");
 
   ErrorF ("-nounicodeclipboard\n"
 	  "\tDo not use Unicode clipboard even if NT-based platform.\n");
--- origsrc/xorg-server-1.6.1.901/hw/xwin/XWin.man	2009-06-24 20:36:20.118598600 -0500
+++ src/xorg-server-1.6.1.901/hw/xwin/XWin.man	2009-06-25 14:42:05.066298600 -0500
@@ -80,10 +80,9 @@ In addition to the normal server options
 manual page, \fIXWin\fP accepts the following command line switches,
 \fIall\fP of which are optional:
 .TP 8
-.B \-clipboard
-Enables the integration
-between the Cygwin/X clipboard and Windows clipboard.  Do not use in
-conjunction with the \fIxwinclip\fP program.
+.B \-[no]clipboard
+Enables [disables] the integration between the Cygwin/X clipboard and 
+Windows clipboard.  The default is enabled.
 .TP 8
 .B "\-clipupdates \fInum_boxes\fP"
 Specify an optional threshold, above which the boxes in a shadow
--- origsrc/xorg-server-1.6.1.901/hw/xwin/winglobals.c	2009-06-24 20:36:18.586075200 -0500
+++ src/xorg-server-1.6.1.901/hw/xwin/winglobals.c	2009-06-25 13:27:37.255238900 -0500
@@ -114,7 +114,7 @@ winDispatchProcPtr	winProcSetSelectionOw
  */
 
 Bool			g_fUnicodeClipboard = TRUE;
-Bool			g_fClipboard = FALSE;
+Bool			g_fClipboard = TRUE;
 Bool			g_fClipboardLaunched = FALSE;
 Bool			g_fClipboardStarted = FALSE;
 pthread_t		g_ptClipboardProc;
--- origsrc/xorg-server-1.6.1.901/hw/xwin/winprocarg.c	2009-06-24 20:36:26.138690600 -0500
+++ src/xorg-server-1.6.1.901/hw/xwin/winprocarg.c	2009-06-25 14:34:01.180136000 -0500
@@ -889,11 +889,23 @@ ddxProcessArgument (int argc, char *argv
    */
   if (IS_OPTION ("-clipboard"))
     {
+      /* Now the default, we still accept the arg for backwards compatibility */
       g_fClipboard = TRUE;
 
       /* Indicate that we have processed this argument */
       return 1;
     }
+
+  /*
+   * Look for the '-noclipboard' argument
+   */
+  if (IS_OPTION ("-noclipboard"))
+    {
+      g_fClipboard = FALSE;
+
+      /* Indicate that we have processed this argument */
+      return 1;
+    }
 #endif
 
 
